js/events/core/emitter_registrator.js

```diff
@@ -230,8 +231,11 @@ var EventManager = Class.inherit({
     }
 });
 
-var eventManager = new EventManager();
 
+var eventManager;
+domAdapter.ready(function() {
+    eventManager = new EventManager();
```

Comment:
**eventManager** is undefined by default. So **eventManager.isActive** can throw an exception.